main.py

```diff
@@ -75,6 +87,21 @@ def main():
                 player_has_card = True
                 print(f"{player.name} discarded {card_on_table} (player left with {len(player.cards)} cards) \n")
                 break
+        if not player_has_card:
+            for card in player.cards:
+                if card.change_color:
+                    player_has_card = True
+                    card_on_table = card
+                    player.cards.remove(card)
+                    print(f"{player.name} discarded {card_on_table} (player left with {len(player.cards)} cards) \n")
+                    if not player.cards:
+                        break
+                    common_color = find_most_common_color(player.cards)
+                    common_color_cards = [card for card in player.cards if card.color == common_color]
+                    for common_card in common_color_cards:
+                        card_on_table = common_card
+                        player.cards.remove(common_card)
```

Comment:
Taki rule: 
after the play changes the color he doesn't drop another card, however, the next play must play with the selected color.